docs/cookbook/adding-a-new-administration-page.md

```diff
@@ -115,6 +114,78 @@ The event subscriber should be auto-discovered by Symfony, reconfiguring the men
 
 ![Dashboard admin menu after the modification](img/dashboard-menu-after.png)
 
+## Access control
+
+When adding a new administrator page, you need to decide which [administrator roles](../administration/admin-rights.md) should have access to it.
+For the new agenda, you will probably need to create a new role. Usually, a `VIEW`/`FULL` pair of roles is used, so let's follow the convention here as well. In `Roles.php`, add the new roles, define theirs translations, and put them into the hierarchy:
+
+```diff
```

Comment:
Almost all content here is added text. What about using `php` instead of `diff`? I think it's a little more clear when reading